README.md

```diff
@@ -104,6 +104,27 @@ The transaction is created when the first SQL statement is executed.
 exits the *with* context and the queries succeed, otherwise
 `trino.dbapi.Connection.rollback()` will be called.
 
+# Extra Credential
+
+Send [`extraCredentials`](https://trino.io/docs/current/installation/jdbc.html?highlight=extra%20credential#parameter-reference)
```

Comment:
The documentation is for JDBC driver and `highlight` looks wrong. I would recommend using https://trino.io/docs/current/develop/client-protocol.html#client-request-headers instead.